Frontend/src/components/Input/PopupFormular.svelte

```diff
@@ -124,7 +126,18 @@
               );
             }
           });
-        dispatch("save");
+          let emptyInputs = popupFormularConfiguration.inputs.filter((input) => {
+            const value = String(
+              $keyValueStore[input.bindTo.keyValueStoreKey][input.bindTo.attr]
+            ).trim();
+            return value.length === 0 && input.required !== false;
```

Comment:
```suggestion
            return value.length === 0 && input.required === true;
```

Reply:
Why? If `required` is missing, then this fails. This isn't my intended behaviour.

Reply:
Sorry my fault